app/models/user.py

```diff
@@ -147,6 +147,13 @@ def activate(self):
         else:
             return self
 
+    def deactivate(self):
+        if self.is_active:
+            user_data = user_api_client.deactivate_user(self.id)
+            return self.__class__(user_data["data"])
```

Comment:
What is being returned here by reaching into the `__class__` property?  Is there a public ("public", I know) property we can reference instead of relying on Python object internals that aren't meant to be accessed/referenced directly?

Reply:
That is a copy and paste job from activate().  I do not know.  I included it because of pattern matching and symmetry.

Reply:
If it's consistent with what's there in the app, then fine for now until we can cleanup more later!